LICENSE

```diff
@@ -309,6 +309,9 @@
    ./dev/ci/chart_schema.yaml
    ./dev/ci/lintconf.yaml
 
+   AWS SDK
+   ./core/src/main/java/org/apache/gravitino/listener/api/info/Either.java
+
```

Comment:
What about the NOTICE file? The AWS SDK has a NOTICE file https://github.com/aws/aws-sdk-java-v2/blob/master/NOTICE.txt, see https://infra.apache.org/licensing-howto.html#bundle-asf-product.

Reply:
Thanks. I appended the NOTICE. Maybe you can take a look when you are free.